h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -366,7 +366,8 @@ public synchronized TwoDimTable to2dTable() {
   public int nclasses() {
     if (_parms._family == Family.multinomial || _parms._family == Family.ordinal)
       return _nclass;
-    if (_parms._family == Family.binomial || _parms._family == Family.quasibinomial)
+    if (_parms._family == Family.binomial || _parms._family == Family.quasibinomial 
+            || _parms._family.equals(Family.fractionalbinomial))
```

Comment:
```suggestion
    if (Family.binomial == _parms._family || Family.quasibinomial == _parms._family
            || Family.fractionalbinomial == _parms._family)
```

Comparison done in a coherent way. Just for better readability.